static/styles/alerts.scss

```diff
@@ -1,11 +1,32 @@
-/* general alert styling changes */
-.alert {
+$alert-red: hsl(16, 60%, 45%);
+$alert-error-red: hsl(0, 80%, 40%);
+
+.alert-base {
     display: none;
 
     &.show {
         display: block;
+
+        animation-name: fadeIn;
+        animation-duration: 0.3s;
+        animation-fill-mode: forwards;
     }
 
+    &.fade-out {
+        animation-name: fadeOut;
+        animation-duration: 0.3s;
+        animation-fill-mode: forwards;
+    }
+
+    .faded {
+        opacity: 0.7;
+    }
+}
```

Comment:
Can you split the extraction of `alert-base` into a prep refactor commit?  That'd make it easier to see the actual code changes.